swap_database.yml

```diff
@@ -96,10 +96,12 @@
         state: touch
     # The replicant service should still be stopped at this point.
```

Comment:
Hmm, for non-single-server environments, doesn't the postgresql service need to be restarted on replicant?